repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/report-main.ftl

```diff
@@ -2,14 +2,40 @@
 <#include "../../include/imports.ftl">
 <#include "../../include/page-meta-data.ftl">
 <#import "../macros/components.ftl" as hee>
+<#include "../utils/document-formats.ftl">
 
 <@hst.setBundle basename="uk.nhs.hee.web.global"/>
 
+<#macro docDetailBlockForDocLink docLink>
+    <@hst.link var="fileURL" hippobean=docLink>
+        <@hst.param name="forceDownload" value="true"/>
+    </@hst.link>
+    <#assign docType>${docLink.filename?keep_after_last(".")}</#assign>
+    <li>
+        <a class="nhsuk-resources__link" href="${fileURL}" title="${docLink.filename}  (opens in new window)">
+            ${docLink.filename?keep_before_last(".")} - <span class="nhsuk-resources__tag nhsuk-resources__${docType}">${getDocumentFormat(docLink.filename?keep_after_last("."))}</span>
+        </a>
+        <#setting number_format="0.##">
+        <span class="nhsuk-resources__docSize">
+            <#if docLink.lengthMB < 0.3>
```

Comment:
Do we have a requirement for displaying the file size (as I am struggling to find one in either the ticket/spec) ? Also, I couldn't find the file size display in the right hand side boxes.